pkl-core/src/main/java/org/pkl/core/ast/member/ObjectMember.java

```diff
@@ -27,6 +27,9 @@
 import org.pkl.core.util.Nullable;
 
 public final class ObjectMember extends Member {
+
+  private final boolean isInIterable;
```

Comment:
Seems costly... not worth rolling it into the member's `modifiers`? Probably worth profiling (both `jpkl` and native) on large corpora.

Reply:
Good idea! Will make this a modifier instead.